### ISSUE TITLE
open-vm-tools: error suspending virtual machine

### ISSUE THREAD

**Issue by jabenninghoff** [rancher/os#1377]
**RancherOS Version: (ros os version)**
ros version v0.7.0

**Where are you running RancherOS? (docker-machine, AWS, GCE, baremetal, etc.)**
VMware Fusion

With `open-vm-tools` enabled and running, suspending the RancherOS generates the following error: "The request to Suspend this virtual machine failed because the corresponding VMware Tools script did not run successfully." Resuming the machine generates a similar error.

Suspending the machine a second time is successful.

From the thread on https://gist.github.com/drakkhen/5473067, this is caused by an error in the shutdown script. Manually testing the suspend-vm-default script within the running open-vm-tools docker container fails with a (fatal) error:
```
Tue Nov 8 03:02:29 UTC 2016 : Executing '/etc/vmware-tools/suspend-vm-default'
Tue Nov 8 03:02:29 UTC 2016 : Executing '/etc/vmware-tools/scripts/vmware/network'

Nov 08 03:02:29 network: ip and ifconfig not in search path.
```
As a workaround, *removing* `/etc/vmware-tools/scripts/vmware/network` from the  image fixes the issue.

**Comment by SvenDowideit** [comment]
ah yup, implied dependencies based on non-container OS installs :/

lets see if i can quickly fix that too

**Comment by SvenDowideit** [comment]
@jabenninghoff I've added ip and ifconfig to the `rancher/os-openvmtools:v0.8.0-rc9` image

can you please try a new v0.8.0-rc9 setup? it seems to make suspend work for me, but my vmware workstation on windows home isn't exactly reliable.

**Comment by jabenninghoff** [comment]
Hi @SvenDowideit, thanks, I tested v0.8.0-rc9, fixed!

**Comment by jabenninghoff** [comment]
Ok @SvenDowideit I spoke to soon - I can suspend and resume without getting an error, but suspend/resume now causes my VM to drop off the network. I did some checking, and after resuming eth0 has dropped its ip(v4) address:

eth0 before:
```
eth0      Link encap:Ethernet  HWaddr 00:0C:29:68:FA:81  
          inet addr:192.168.1.31  Bcast:0.0.0.0  Mask:255.255.255.0
          inet6 addr: fe80::20c:29ff:fe68:fa81/64 Scope:Link
          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1
          RX packets:2477 errors:0 dropped:0 overruns:0 frame:0
          TX packets:2094 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1000 
          RX bytes:1459996 (1.3 MiB)  TX bytes:635441 (620.5 KiB)
``` 

eth0 after:
```
eth0      Link encap:Ethernet  HWaddr 00:0C:29:68:FA:81  
          inet6 addr: fe80::20c:29ff:fe68:fa81/64 Scope:Link
          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1
          RX packets:2692 errors:0 dropped:0 overruns:0 frame:0
          TX packets:2298 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1000 
          RX bytes:1544575 (1.4 MiB)  TX bytes:686687 (670.5 KiB)
```

I'm wondering if the workaround (removing `/etc/vmware-tools/scripts/vmware/network`) is a better permanent fix.

**Comment by SvenDowideit** [comment]
cool - I'll try that tomorrow. frustrating :)

**Comment by jabenninghoff** [comment]
Hi @SvenDowideit I did some more research and while removing the script will work, I think I found the problem: the logic in the script first tries `ifdown`, and if that fails, it calls `ip` and *removes the address from the interface*. I think adding `ifup` and `ifdown` to the image in addition to `ip` and `ifconfig` is probably a better fix.

I'd argue that removing the addresses from the interface is a bad design decision on the part of VMware, but that's a different discussion.

**Comment by janeczku** [comment]
I think the solution is to add custom suspend/resume scripts like here to the image: https://gist.github.com/anonymous/961e73a82b3629c94179
Probably just a NOOP script, idk if the networking stuff is required...

**Comment by SvenDowideit** [comment]
ok, I've updated the `rancher/os-openvmtools:v0.8.0-rc9` image to also include ifdown
I'll test it later when I get back to having ok net

**Comment by SvenDowideit** [comment]
@jabenninghoff I presume this was resolved in 0.8.0?
